Services/DataBaseService/Repositories/UserRepository.cs

```diff
@@ -54,16 +54,19 @@ public User GetUserById(Guid userId)
         public void DeleteUser(Guid userId)
         {
             var dbUserCredential = dbContext.UsersCredentials.FirstOrDefault(uc => uc.UserId == userId);
-            if (dbUserCredential == null)
+
+            if (dbUserCredential is null)

```

Comment:
How work `is`

Reply:
> How work `is`

As i know it is literally syntactic sugar for == null